packages/sirius-web/frontend/sirius-web-application/src/views/project-settings/ProjectSettingsView.types.ts

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+export interface ProjectSettingsParams {
+  projectId: string;
+}
+export interface ProjectSettingTabContribution {
+  title: string;
+  component: () => JSX.Element | null;
+}
```

Comment:
You will also need a path here like 'images' or something for the ProjectImageSettings in order to give us the ability to reference a setting page directly.

Reply:
An icon will also be necessary